cypress/e2e/basic.cy.js

```diff
@@ -0,0 +1,10 @@
+describe('empty spec', () => {
```

Comment:
Maybe we could rename this to just `sample test`?

Reply:
good call, changed!